packages/@lwc/integration-karma/test/template/attribute-namespace/index.spec.js

```diff
@@ -0,0 +1,17 @@
+import { createElement } from 'lwc';
+import Component from 'x/component';
+
+describe('attribute with namespace', () => {
+    it('correctly renders xml:* and xlink:* attributes', async () => {
+        const elm = createElement('x-component', { is: Component });
+        elm.title = 'foo';
+        elm.lang = 'en-US';
+        document.body.appendChild(elm);
+        await Promise.resolve();
+
+        // svg namespace
+        expect(elm.shadowRoot.querySelector('image').getAttribute('xlink:title')).toBe('foo');
+        // xml namespace
+        expect(elm.shadowRoot.querySelector('label').getAttribute('xml:lang')).toBe('en-US');
+    });
```

Comment:
The `xml:*` namespace behavior was untested before, so I added a test. SVG was already tested elsewhere due to `xlink:href` but I added another one here for good measure.

Reply:
BTW I thought `getAttributeNS` would work, but I couldn't actually get it working? And the `xlink:href` test elsewhere also used `getAttribute`. 🤷